modules/tests/test/create_vm_from_manifest_test.go

```diff
@@ -242,4 +242,71 @@ var _ = Describe("Create VM from manifest", func() {
 			"vm.kubevirt.io/name": vmName,
 		}))
 	})
+
+	Context("with StartVM", func() {
+		table.DescribeTable("VM is created successfully", func(config *testconfigs.CreateVMTestConfig, running bool) {
```

Comment:
do we have to create a new table for this?

Reply:
The tests looks better when they are structured in different tables / contexts

Reply:
ok, I don't like repeating the test logic, but it is matter of preference